core/src/test/scala/integration/kafka/api/CustomQuotaCallbackTest.scala

```diff
@@ -89,6 +89,7 @@ class CustomQuotaCallbackTest extends IntegrationTestHarness with SaslSetup {
     producers.clear()
     consumers.foreach(_.close(0, TimeUnit.MILLISECONDS))
     consumers.clear()
+    adminClients.foreach(_.close())
```

Comment:
Nit:  not sure why we're clearing the producer and consumer lists. Do we want to do the same here?

Reply:
`producers` and `consumers` are defined in the superclass and they are closed in the superclass `tearDown`. For this test, we close them here to avoid blocking for throttled requests to complete. The `clear()` is to avoid calling `close()` again, but I could remove that if we think it is confusing since a second `close()` should be a no-op. `adminClients` is defined in this class, so `clear()` is unnecessary.